src/elements/emby-slider/emby-slider.css

```diff
@@ -1,4 +1,4 @@
-_:-ms-input-placeholder {
+:-ms-input-placeholder {
```

Comment:
eh?

Reply:
Old browser hack for things we don't support: http://browserhacks.com/#hack-f1070533535a12744a0381a75087a915